### ISSUE TITLE
[CR]: Working Clock

### ISSUE THREAD

**Issue by FabDonRixos** [FabDonRixos/awesome-arts#16]
### Category

Complex

### Inspiration

Time

### Description

I would like to have a working analog clock with 3 hands that ticks in the right time delation.

### Are there any possible issues with the idea

Getting the current time in css only might be difficult/impossible so i would suggest to just define an exact time and the clock will tick from there.

### TODO's

- [X] The component hasn't been suggested / isn't already implemented yet.
- [X] The title is clear and descriptive.
- [X] The component is properly described.
- [X] The Component has good tags.

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by duyb2111918** [comment]
hey @FabDonRixos  can you assign me this issue? I want to try it out.

**Comment by FabDonRixos** [comment]
@duyb2111918 Sure, best of luck.

**Comment by duyb2111918** [comment]
> @duyb2111918 Sure, best of luck.

Thank! @FabDonRixos, but I have some questions about it. Can I use some other language like JS combined with it or do I only use CSS to implement the component? And which directory should my components be stored in? Please respond as soon as possible so I can implement it as quickly as possible

**Comment by FabDonRixos** [comment]
Hi @duyb2111918
I got a mail where you asked a question but i dont see it in github.
Maby you deleted it on porpouse or the system is weard but anyways, if you havent figured out yet:
Yes the point is to only use CSS/SCSS for the component and everything you need for the contribution should be written under Contributing to `Awesome-Arts` inside the ReadMe.md

**Comment by duyb2111918** [comment]
> Hi @duyb2111918 I got a mail where you asked a question but i dont see it in github. Maby you deleted it on porpouse or the system is weard but anyways, if you havent figured out yet: Yes the point is to only use CSS/SCSS for the component and everything you need for the contribution should be written under Contributing to `Awesome-Arts` inside the ReadMe.md

Yes, actually I posted the question to the system but then I rechecked it and understood it so I removed it. Anyway thanks for your feedback on my question, I'm on my way to implementing it

**Comment by duyb2111918** [comment]
@FabDonRixos I have a question I created a working clock in the artwork folder using a template and now I had to create a component for it or not. Please help me to figure it out. I'm very sorry that there are many unknowns because this is the first time I have solved an issue.

**Comment by FabDonRixos** [comment]
@duyb2111918 No worry's you can't ask to many questions.
I'm not quite sure if I understood you question right, but you should create a folder named for example clock in there there should be a "Clock.tsx" and a "module.scss" and inside the tsx you shall import the scss and you have to create a new default export function component.

export default function Clock() {

 return (
 // your code
 );
}

**Comment by duyb2111918** [comment]
Y

> @duyb2111918 No worry's you can't ask to many questions. I'm not quite sure if I understood you question right, but you should create a folder named for example clock in there there should be a "Clock.tsx" and a "module.scss" and inside the tsx you shall import the scss and you have to create a new default export function component.
> 
> export default function Clock() {
> 
> return ( // your code ); }

Thank you for replying! Yes, I already done it but i wondering that in folder components of src folder do i had to create or do anything on it?

**Comment by FabDonRixos** [comment]
@duyb2111918 Oh Great, thanks for your feedback, indeed this could be misunderstanding and I will have to adjust this. 
And to you question no in the components folder you shall not do anything for a new component, the folder artwork is for all components and the components folder is for all components used to make the overall website.
Sorry for the confusion.

**Comment by FabDonRixos** [comment]
@duyb2111918 Hi there

Friendly reminder: I just wanted to reminde you that you still have an open pull request with your cool clock you build somewhere around November. If you currently don’t have time or desire to work on it that’s no problem we have plenty of time. But it would be a shame if it would never see sunlight. 

Best Regards
And also a Happy new Year
